modules/platforms/dotnet/Apache.Ignite.Tests/ExceptionsTests.cs

```diff
@@ -52,13 +52,15 @@ public void TestExceptionsAreSerializableAndHaveRequiredConstructors()
                 var stream = new MemoryStream();
                 var formatter = new BinaryFormatter();
 
+#pragma warning disable SYSLIB0011 // BinaryFormatter is obsolete
```

Comment:
Is it anything to be used instead?

Reply:
Reworked this test to avoid obsolete `BinaryFormatter`.

NOTE: The future of `ISerializable` and related types is not clear, they may become obsolete too (see https://github.com/dotnet/runtime/issues/70208). But for now we follow existing guidelines and implement this interface properly.

Reply:
👍 nice